cordformation/src/main/kotlin/net/corda/plugins/NetworkParameterOverrides.kt

```diff
@@ -1,29 +1,58 @@
 package net.corda.plugins
 
-import com.typesafe.config.Config
-import com.typesafe.config.ConfigFactory
-import com.typesafe.config.ConfigObject
-import com.typesafe.config.ConfigValueFactory
+import com.typesafe.config.*
 import org.gradle.api.Action
 import org.gradle.api.NamedDomainObjectContainer
 import org.gradle.api.Project
+import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.Nested
+import org.gradle.api.tasks.Optional
+import java.time.Duration
 import javax.inject.Inject
 
 open class NetworkParameterOverrides @Inject constructor(project: Project) {
 
+    @get:Optional
+    @get:Input
+    var minimumPlatformVersion: Int? = null
+
+    @get:Optional
+    @get:Input
+    var maxMessageSize: Int? = null
+
+    @get:Optional
+    @get:Input
+    var maxTransactionSize: Int? = null
+
     @get:Nested
     val packageOwnership: NamedDomainObjectContainer<PackageOwnership> = project.container(PackageOwnership::class.java)
 
     fun packageOwnership(action: Action<in NamedDomainObjectContainer<PackageOwnership>>) {
         action.execute(packageOwnership)
     }
 
+    @get:Optional
+    @get:Input
+    var eventHorizon: Duration? = null
+
+    fun isEmpty() = minimumPlatformVersion == null &&
```

Comment:
This `isEmpty()` function is missing the `eventHorizon` property. You can check whether a "lazy" property has been assigned a value using `isPresent`.